packages/safelink/package.json

```diff
@@ -26,7 +26,8 @@
     "es"
   ],
   "dependencies": {
-    "@ndla/icons": "^1.4.1"
+    "@ndla/icons": "^1.4.1",
+    "@ndla/button": "^1.1.1"
```

Comment:
Skulle denne vært 1.1.5 nå?

Reply:
Ser ellers bra ut for meg, men får dessverre ikke lov til å approve en PR jeg selv har opprettet